.github/CODEOWNERS

```diff
@@ -277,6 +299,8 @@
 /sdk/mixedreality/azure-mixedreality-authentication/                 @RamonArguelles
 /sdk/remoterendering/                                                @FlorianBorn71
 
+# AzureSDKOwners: @kristapratico
```

Comment:
Our team owns OpenAI and should be responsible for issues coming in for it?   If Python is like the other languages, the service team owns the library and we're just supporting them.   

If that's the case, we'll want to remove `AzureSDKOwners` in favor of `ServiceOwners`.
Even if our team owns the OpenAI issues, we'll want a `Service Owners` here (or a not-in-repo entry below) to ensure that "Service Attention" works properly.